README.md

```diff
@@ -1,4 +1,4 @@
-# <a href="https://github.com/TheAlgorithms/">The Algorithms</a> – Nim&nbsp;<img src="https://raw.githubusercontent.com/nim-lang/assets/master/Art/logo-crown.png" height="28px"/><img align="right" src="https://raw.githubusercontent.com/TheAlgorithms/website/main/public/logo.svg" height="100">
+# <a href="https://github.com/TheAlgorithms/">The Algorithms</a> – Nim&nbsp;<img src="https://raw.githubusercontent.com/nim-lang/assets/master/Art/logo-crown.png" height="28px"/><a href="https://github.com/TheAlgorithms/"><img align="right" src="https://raw.githubusercontent.com/TheAlgorithms/website/main/public/logo.svg" height="100"></a>
```

Comment:
The image already links to the website, not the Github page. Why do you add a link here?

Reply:
Before this change it linked to the image file itself, not like it does at the current state of the Readme.